src/files/FilesPage.js

```diff
@@ -35,17 +42,17 @@ class FilesPage extends React.Component {
     doUpdateHash(`/files${link}`)
   }
 
-  onInspect = (hash) => {
+  onInspect = ([file]) => {
```

Comment:
Why add the array here?

PS: I'm dropping in without much background.

Reply:
We were only sending the required info for each hook: the delete only needed paths, the inspect only needed the hash, etc.

But since now we also need the type of the file, we would have to send two different things for delete hook.

I then decided just to send everything to every hook and the hook uses the data it wants. You can see the changes on [this commit](https://github.com/ipfs-shipyard/ipfs-webui/pull/689/commits/f4e5800f8997ae3cc09df9a2534e9e13aa53302b).

Reply:
Sounds good, thank you 👍